src/core/components/libp2p.js

```diff
@@ -20,7 +20,14 @@ module.exports = function libp2p (self) {
           webRTCStar: get(config, 'Discovery.webRTCStar.Enabled'),
           bootstrap: get(config, 'Bootstrap'),
           dht: get(self._options, 'EXPERIMENTAL.dht'),
-          modules: self._libp2pModules
+          modules: self._libp2pModules,
+          relay: {
+            enabled: get(config, 'EXPERIMENTAL.Relay.Enabled', false),
+            hop: {
+              enabled: get(config, 'EXPERIMENTAL.Relay.HOP.Enabled', false),
+              active: get(config, 'EXPERIMENTAL.Relay.HOP.Active', false)
```

Comment:
What is the difference between Enabled and Active?

Reply:
`Active` is for active/passive relaying, `Enabled` is just the flag.

Reply:
Is this matching go-ipfs config?